### PR TITLE
Fixed download flyer link being covered by banner

### PR DESCRIPTION
**Pull request by danthaman44**
Verified "Download our flyer" link is clickable on desktop and mobile

Screen Shot 2020-09-10 at 8 47 57 PM
Screen Shot 2020-09-10 at 8 48 11 PM

### CODE REVIEW

File: src/components/MerchantsPage/styles.module.scss [hunk 1]
```diff
@@ -152,6 +152,39 @@
   }
 }
 
+.bannerContentContainer {
+  // grid relative to parent container
+  grid-row: 1;
+  grid-column: 1;
+  z-index: 1;
```
Comment (justintmckibben) [inline src/components/MerchantsPage/styles.module.scss:159]: Why do we need to change the z-index?

File: src/components/MerchantsPage/styles.module.scss [hunk 2]
```diff
@@ -152,6 +152,39 @@
...
+
+  // creating grid container for its own children components
+  display: grid;
```
Comment (justintmckibben) [inline src/components/MerchantsPage/styles.module.scss:162]: Why not use flexbox?

**Review by justintmckibben — APPROVED** [review]

File: src/components/MerchantsPage/styles.module.scss (hunk 1 above)
Comment (danthaman44) [reply src/components/MerchantsPage/styles.module.scss:159]: Not setting a z-index causes the content to get hidden behind the banner 😬

File: src/components/MerchantsPage/styles.module.scss (hunk 2 above)
Comment (danthaman44) [reply src/components/MerchantsPage/styles.module.scss:162]: I 100% copied this from the NYC Map banner implementation. If I did it again from scratch, I would definitely consider flexbox.

File: src/components/MerchantsPage/styles.module.scss [hunk 3]
```diff
@@ -152,6 +152,39 @@
   }
 }
 
+.bannerContentContainer {
```
Comment (aliceluo2048) [inline src/components/MerchantsPage/styles.module.scss:155]: Is the css exactly the same as `contentContainer` except you have a `z-index: 1` and you added   `@include for-phone-only {
    height: 480px;`

Comment (danthaman44) [reply src/components/MerchantsPage/styles.module.scss:155]: Yes 😬

Comment (danthaman44) [reply src/components/MerchantsPage/styles.module.scss:155]: You are welcome to refactor my code at any time, happy to review the PR :)

Comment (aliceluo2048) [reply src/components/MerchantsPage/styles.module.scss:155]: Is contentContainer even being used then...

File: src/components/MerchantsPage/styles.module.scss (hunk 1 above)
Comment (justintmckibben) [reply src/components/MerchantsPage/styles.module.scss:159]: This is a bit of code smell 😬

File: src/components/MerchantsPage/styles.module.scss (hunk 3 above)
Comment (aliceluo2048) [reply src/components/MerchantsPage/styles.module.scss:155]: > You are welcome to refactor my code at any time, happy to review the PR :)

Yeah nah

**Pull request merged**